client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepContainer from '../step-container';
+import { hasNonEmptyLeaves } from 'woocommerce/woocommerce-services/lib/utils/tree';
+import { toggleStep, confirmCustoms } from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+	isCustomsFormStepSubmitted,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import StepConfirmationButton from '../step-confirmation-button';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+
+const CustomsStep = ( props ) => {
+	const {
+		errors,
+		expanded,
+		translate,
+		isSubmitted,
+		dummyFieldChecked,
+	} = props;
+	const summary = hasNonEmptyLeaves( errors ) ? translate( 'Customs information incomplete' ) : translate( 'Customs information valid' );
+
+	return (
+		<StepContainer
+			title={ translate( 'Customs' ) }
+			summary={ isSubmitted ? summary : '' }
+			expanded={ expanded }
+			toggleStep={ props.toggleStep }
+			isSuccess={ isSubmitted && ! hasNonEmptyLeaves( errors ) }
+			isError={ isSubmitted && hasNonEmptyLeaves( errors ) } >
+			<Checkbox checked={ Boolean( dummyFieldChecked ) } onChange={ props.toggleDummyField } /> Check this to make validation pass.
+			<StepConfirmationButton
+				disabled={ hasNonEmptyLeaves( errors ) }
+				onClick={ props.confirmCustoms } >
+				{ translate( 'Confirm' ) }
+			</StepConfirmationButton>
+		</StepContainer>
+	);
+};
+
+CustomsStep.propTypes = {
+	siteId: PropTypes.number.isRequired,
```

Comment:
Let's make sure to add the rest of the props - e.g. `errors`, `expanded`, `toggleStep`, etc.

Reply:
I don't think this is in writing anywhere, but the way we've usually use `PropTypes` is to only declare the props that the component receives from its parent. The props that are computed using `connect()` don't need to be documented here because they are computed and added in this same file, and also another component that wants to render this one doesn't need to know about those. Do you agree with that?